app.js

```diff
@@ -0,0 +1,29 @@
+// Require Express
+const express = require('express');
+
+// Express server handling requests and responses
+const app = express();
+
+// Make everything inside of public/ available
+app.use(express.static('public'));
+
+// home route:
+app.get('/', (request, response, next) => response.sendFile(__dirname + '/views/index.html'));
```

Comment:
Esta forma está bien, pero mejor que utilicéis esta otra, ya que con render puedes pasar parámetros :

app.get('/', (req, res, next) => res.render('index'))

En este ejemplo ves que está con req, res y next, te acostumbrarás a verlo así.

Reply:
Corregido. Gracias!!